tests/rust/struct.rs

```diff
@@ -28,11 +28,18 @@ struct TupleRenamed(i32, f32);
 #[repr(C)]
 struct TupleNamed(i32, f32);
 
+#[repr(C)]
+struct WithFlexibleArrayMember {
+    x: i32,
+    y: [i8; 0],
```

Comment:
Can you test with some other field afterwards too? Not sure what the behavior there should be, maybe it's fine.

Reply:
I had to rework the whole approach. I've added a specific type to represent flexible array members, and that makes everything much more consistent.
I'm unsure about the idiomaticity of my rust code, don't hesitate to bash me :-)